tox.ini

```diff
@@ -57,6 +57,7 @@ envlist =
 # gevent 1.0 is not python 3 compatible
     gevent_contrib-{py27}-gevent{10}
     httplib_contrib-{py27,py34,py35,py36}
+    jinja2_contrib-{py27,py34,py35,py36}-jinja{210}
```

Comment:
We should probably add tests for `jinja{20,21,22,23,24,25,26,27,28,29,210}`

We like to test most recent patch of every minor version.

Although, this could be a bit overkill since the API has mostly stayed the same. @Kyle-Verhoog opinions?